### DISCUSSION TITLE
Help! For the Education community team, Unable to access copilot

### DISCUSSION THREAD

**Discussion by NoamGolda** [community/community#127892] [GitHub Education]
### Hi everyone,

![image](https://github.com/community/community/assets/84135608/7cb91a99-c3ce-409e-873b-f5763f16cd75)
As seen here^ 
But in VScode extension:
![image](https://github.com/community/community/assets/84135608/4fc21da7-02a2-4838-99c6-3a64c53638c8)

I tried to refresh and uninstall many times but was unable to solve the problem, requesting help

**Comment by davevad93** [comment]
Hi @NoamGolda ,

If you haven't done it yet, try logging out from your Copilot account in VS Code and then log in again, there may be an issue with the authentication token. If that doesn't work either you should contact support by opening a ticket [here](https://support.github.com/contact/account).
